packages/react-core/src/components/NotificationDrawer/__tests__/NotificationDrawerListItemBody.test.tsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
+import { render, screen } from '@testing-library/react';
 import { NotificationDrawerListItemBody } from '../NotificationDrawerListItemBody';
-import { render } from '@testing-library/react';
-import { shallow } from 'enzyme';
 
-test('renders with PatternFly Core styles', () => {
-  const view = render(<NotificationDrawerListItemBody />);
-  expect(view.container).toMatchSnapshot();
-});
+describe('NotificationDrawerListItemBody', () => {
+  test('renders with PatternFly Core styles', () => {
+    render(<NotificationDrawerListItemBody data-testid="test-id" />);
+    expect(screen.getByTestId('test-id').outerHTML).toMatchSnapshot();
+  });
 
-test('className is added to the root element', () => {
-  const view = shallow(<NotificationDrawerListItemBody className="extra-class" />);
-  expect(view.prop('className')).toMatchSnapshot();
-});
+  test('className is added to the root element', () => {
+    render(<NotificationDrawerListItemBody className="extra-class" data-testid="test-id" />);
+    expect(screen.getByTestId('test-id').className).toMatchSnapshot();
+  });
 
-test('list item body with timestamp property applied ', () => {
-  const view = render(<NotificationDrawerListItemBody timestamp="5 minutes ago" />);
-  expect(view.container).toMatchSnapshot();
+  test('list item body with timestamp property applied', () => {
+    render(<NotificationDrawerListItemBody timestamp="5 minutes ago" data-testid="test-id" />);
+    expect(screen.getByTestId('test-id').className).toMatchSnapshot();
+  });
```

Comment:
Suffers the same issue as other NotificationDrawer tests where it should be comparing the outerHTML to the snapshot I believe.